### ISSUE TITLE
DWYL or dwyl?

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/brand#1]
We're currently using a mix of uppercase and lowercase versions of our company name. When creating the new [Facebook Page](https://www.facebook.com/dwylhq) I was unable to set the name as `dwyl`, the rules only allowed me to set it as `Dwyl` or `DWYL`, so I chose the latter.

We need to have a consistent brand and use consistent casing. Which do people prefer? Reasons for and against?

I've outlined some arguments below:

`dwyl`
+currently most widely used
+a more relaxed, informal feel
-is less clear that it is an acronym
-autocapitalises to Dwyl, risks brand-inconsistency with those who don't know it should be lowercase or platforms such as Facebook, which don't accept it

`DWYL`
+makes it more clear that it is an acronym
+consistent letter height (potentially better for branding and merchandise)
+more standard, more professional
-potential to appear less friendly (arguable that the font style could remedy this)

Given the above, I would suggest `DWYL` as the more appropriate of the two. Thoughts?

**Comment by nelsonic** [comment]
my vote is for @dwyl (_lowercase_)
especially when Twitter gives us our freakin' handle!! https://github.com/dwyl/hq/issues/96

**Comment by iteles** [comment]
I find lowercase is more in keeping with our character. 

We spoke earlier today about having a 3 month trial with a hypothesis and finding something measurable for this.

**Comment by markwilliamfirth** [comment]
Would be interesting to somehow quantify customer feedback on this

**Comment by markwilliamfirth** [comment]
<img width="127" alt="screen shot 2017-02-15 at 10 52 14" src="https://cloud.githubusercontent.com/assets/5732209/22971219/ec9aec20-f36c-11e6-9126-fcec2c5bb9ce.png">

Received an email from someone - even though my email clearly says "dwyl" they capitalised the first letter (even though it's not at the start of a sentence). How often does this happen to you @iteles ?

**Comment by nelsonic** [comment]
@markwilliamfirth I don't think we can `expect` people replying to our emails to "_get it **right**_" ... 
we will _inevitably_ see Dwyl, DWYL, _and_ dwyl. and that's ***100% OK***. 👍 

But since we are going to refer to _ourselves_ as "dwyl" or [@dwyl](https://github.com/dwyl/hq/issues/96) or eventually [**dwyl.com**](https://github.com/dwyl/hq/issues/118) (_always lowercase_) I have _faith_ that ***eventually*** people will "_get the message_" ...

## Conclusion (_Proposal_)

We should ***use lowercase*** `dwyl` in **all marketing/branding**.
and `DWYL` or `DWYL LTD` **uppercase** in **legal documents**. 

For now, I don't think we _mind_ how _other_ people write it.

> Quoting FallOutBoy: 
> "_I don't care what you think as long as it's about me..._" https://youtu.be/MRnVBEjeUZE?t=3m1s

i.e. we should only _think_ about this once people are _using_ the product and talking/writing about us.

I'm _far_ more interested in seeing a metric/google alert for how many times people mention or search for dwyl or `#dwyl` ... 📈 

## But it's an Acronym ... shouldn't it be Uppercase...?

***Yes***, `dwyl` is an [**acronym**](https://en.wikipedia.org/wiki/Acronym) and people can _choose_ to expand out the letters when they are thinking of it. 
But as a _brand_ we _prefer_ to _think_ of ourselves as lowercase.


## tl;dr > _Other_ Examples of `lowercase` logos<sup>1</sup>

![patagonia-logo](https://cloud.githubusercontent.com/assets/194400/22974714/8af656bc-f37b-11e6-9b81-836d1af24251.jpg)
![hp logo](https://cloud.githubusercontent.com/assets/194400/22973328/24b35cf2-f375-11e6-96cb-2b30ce958c45.png)
![xerox logo](https://cloud.githubusercontent.com/assets/194400/22973342/31c5e720-f375-11e6-97d8-149db3b71c73.png)
![skype](https://cloud.githubusercontent.com/assets/194400/22973349/39e550ee-f375-11e6-9454-72ec14ab9d57.png)
![citibank logo](https://cloud.githubusercontent.com/assets/194400/22973360/410f7426-f375-11e6-942f-22730b0a8487.png)
![intel logo](https://cloud.githubusercontent.com/assets/194400/22973420/85c47c38-f375-11e6-86a0-8e4082e3cbef.png)
![airbnb](https://cloud.githubusercontent.com/assets/194400/22973442/9406283c-f375-11e6-88dc-c9f44a9d966e.png)
![amazon logo](https://cloud.githubusercontent.com/assets/194400/22974095/b62dfffe-f378-11e6-8b0b-f2724fb2023c.png)
![bing logo](https://cloud.githubusercontent.com/assets/194400/22973475/aed40206-f375-11e6-9599-915327db144d.png)
![twitter logo](https://cloud.githubusercontent.com/assets/194400/22973488/c2301178-f375-11e6-91f4-e9b5af954a30.png)
![motorola logo](https://cloud.githubusercontent.com/assets/194400/22974369/e37c7728-f379-11e6-9f8f-0164ec59aa61.png)
![lenovo logo](https://cloud.githubusercontent.com/assets/194400/22974380/f13cbf1c-f379-11e6-8205-71797750aef9.png)
although: https://goo.gl/keRoon 😮 
![flickr logo](https://cloud.githubusercontent.com/assets/194400/22973502/cfc2f396-f375-11e6-844e-dddea48c34bf.png)
![zipcar](https://cloud.githubusercontent.com/assets/194400/22973631/6bc5cb4c-f376-11e6-9892-df4b8a2f690c.png)
![waze logo](https://cloud.githubusercontent.com/assets/194400/22973750/eed9ba5c-f376-11e6-92f0-66976e7bfd26.png)
![lendlease logo](https://cloud.githubusercontent.com/assets/194400/22974520/9328bd4e-f37a-11e6-9c62-3809e7f0fb86.png)
![mace logo](https://cloud.githubusercontent.com/assets/194400/22974539/abc4e67a-f37a-11e6-8a96-53da1dbbcc78.png)
![snapchat](https://cloud.githubusercontent.com/assets/194400/22973836/486c945e-f377-11e6-8367-812659427fa4.png)
![zendesk](https://cloud.githubusercontent.com/assets/194400/22974238/48c10d5c-f379-11e6-9bad-f6217f0a36f1.png)
![facebook logo iterations](https://cloud.githubusercontent.com/assets/194400/22973716/c7e2e860-f376-11e6-9d7a-1264aa1bff16.png)
![yelp logo](https://cloud.githubusercontent.com/assets/194400/22973800/14e2e37c-f377-11e6-9b88-7fa9b9a78454.png)
![salesforce logo](https://cloud.githubusercontent.com/assets/194400/22974060/7db73910-f378-11e6-8b10-ae714450c2f2.png)
![nest logo](https://cloud.githubusercontent.com/assets/194400/22973928/cc83f980-f377-11e6-8a01-22c328f237c3.png)
![twitch](https://cloud.githubusercontent.com/assets/194400/22974119/d5cacca2-f378-11e6-9622-9e45845bbe0c.png)
![vodafone](https://cloud.githubusercontent.com/assets/194400/22974287/8ba427bc-f379-11e6-9db7-f4b3b98253c2.png)
![udemy-logo-small](https://cloud.githubusercontent.com/assets/194400/22974988/cfbb2664-f37c-11e6-8640-60c88dd8ee7c.png)
![xero logo](https://cloud.githubusercontent.com/assets/194400/22974316/a8ca22e2-f379-11e6-92b3-16de26483c46.png)
![vimeo_logo](https://cloud.githubusercontent.com/assets/194400/22975336/4ca70eee-f37e-11e6-8a9b-1b2a755888a6.jpg)
![docker nodejs logo](https://cloud.githubusercontent.com/assets/194400/22975028/f887b36e-f37c-11e6-8083-aa676c5e11c2.png)
![mark-logo](https://cloud.githubusercontent.com/assets/194400/22974593/fc6bd750-f37a-11e6-9046-2acab2513db5.png)
![asana logo](https://cloud.githubusercontent.com/assets/194400/22974004/2caf2974-f378-11e6-92b3-62f52ad9a7b9.png)

<sup>1</sup>We are _aware_ that ***most*** companies use an Uppercase letter _somewhere_ in their name/logo if not ***all*** of it. we just want to _highlight_ that we aren't the ***only*** ones with this "_crazy_" idea of using lowercase. 👍  😉 

![image](https://cloud.githubusercontent.com/assets/194400/22975395/83894b16-f37e-11e6-8b3a-edefdb7c51d8.png)

## Notes

eBay have _never_ used a capital "e" ... they used to use a capital "B" in their logo:
![ebay capital logo](https://cloud.githubusercontent.com/assets/194400/22972099/19edacaa-f370-11e6-82d9-7d10e1c057bb.png)

they _recently_ changed their logo to all lowercase (_at great expense!_) to simplify things:
![image](https://cloud.githubusercontent.com/assets/194400/22972116/3ccbe502-f370-11e6-9dcc-dc86abda9d5b.png)

but most people still write their name as `eBay`...
e.g: https://www.google.co.uk/search?tbm=nws&q=ebay
and since the stock symbol is `EBAY` most financial press refer to them by their symbol.
Like most companies that get featured in the press they have a guidelines page:
https://brandpermission.ebay.com/guidelines

interesting point: _none_ of the "Top 10" logos are lowercase: https://youtu.be/dQV1b57F8D0?t=8m11s 
and the `#1` (_most recognised_) logo in the world is just a "tick" ("_swoosh_") 
![image](https://cloud.githubusercontent.com/assets/194400/22975757/ab28a594-f37f-11e6-9366-2979862fcd2c.png)

Let's maintain focus on the _product_ for now and we can _evolve_ the branding once people are _using_ the app! ❤️  ✅  🚀

**Comment by iteles** [comment]
@markwilliamfirth I don't feel like it's a big deal despite not being aesthetically 'amazing'. I personally prefer Dwyl to DWYL.

I mentioned it in our verbal conversation but not here so it's worth adding: I don't see 'dwyl' as an acronym in the long term either.

Shall we stick with our plan of testing it for 3 months and taking a checkpoint then? We can use this issue to catalogue when people 'get it wrong' and then we can see if we have cause for concern.

@nelsonic Thanks for the branding research above, great examples!

**Comment by markwilliamfirth** [comment]
This issue should be used to collect cases where people misuse the casing and any qualitative comments associated with it to assess feedback and public perception.

@iteles the original comment was a question to ask if you had any similar evidence of people writing "dwyl" or "DWYL" mistakenly as "Dwyl".

With regards to the comment above I definitely see it as an acronym and I was under the impression all of our contractors and clients thought that too. Maybe it will evolve a few years down the line, but maybe it's not at that stage at the moment?

**Comment by markwilliamfirth** [comment]
@nelsonic I may be mistaken, but I'm under the impression that a lot of or maybe all of those companies write their company name capitalised or in uppercase. Agreed that the logo design is a separate matter to how we write the company name in emails etc.

**Comment by markwilliamfirth** [comment]
<img width="138" alt="screen shot 2017-02-15 at 13 43 03" src="https://cloud.githubusercontent.com/assets/5732209/22977391/64801d82-f386-11e6-913c-6eff92dbdc21.png">
<img width="129" alt="screen shot 2017-02-15 at 13 44 41" src="https://cloud.githubusercontent.com/assets/5732209/22977396/648d671c-f386-11e6-8be8-10722e50c7fd.png">
<img width="203" alt="screen shot 2017-02-15 at 13 45 03" src="https://cloud.githubusercontent.com/assets/5732209/22977394/6488f844-f386-11e6-8825-6839a674682a.png">
<img width="151" alt="screen shot 2017-02-15 at 13 45 17" src="https://cloud.githubusercontent.com/assets/5732209/22977392/64820bce-f386-11e6-8c81-561d80633925.png">
<img width="186" alt="screen shot 2017-02-15 at 13 45 51" src="https://cloud.githubusercontent.com/assets/5732209/22977393/6484fcb2-f386-11e6-9d54-a4546245d321.png">
<img width="96" alt="screen shot 2017-02-15 at 13 46 07" src="https://cloud.githubusercontent.com/assets/5732209/22977395/648c36d0-f386-11e6-9bea-5d9ebf1cb220.png">
<img width="201" alt="screen shot 2017-02-15 at 13 46 24" src="https://cloud.githubusercontent.com/assets/5732209/22977397/6493d0de-f386-11e6-9bf8-bbad130a9a23.png">
<img width="128" alt="screen shot 2017-02-15 at 13 46 41" src="https://cloud.githubusercontent.com/assets/5732209/22977398/649baa70-f386-11e6-805f-c847d0bf8365.png">
<img width="68" alt="screen shot 2017-02-15 at 13 47 02" src="https://cloud.githubusercontent.com/assets/5732209/22977399/649bfa34-f386-11e6-9c41-3aed39c105d9.png">
<img width="180" alt="screen shot 2017-02-15 at 13 47 12" src="https://cloud.githubusercontent.com/assets/5732209/22977402/64a43cee-f386-11e6-9b6d-9a2b1cd763d7.png">
<img width="161" alt="screen shot 2017-02-15 at 13 47 26" src="https://cloud.githubusercontent.com/assets/5732209/22977400/64a1f074-f386-11e6-8bb5-940881f41314.png">
<img width="125" alt="screen shot 2017-02-15 at 13 47 38" src="https://cloud.githubusercontent.com/assets/5732209/22977401/64a373e0-f386-11e6-99c9-5289f159560a.png">
<img width="105" alt="screen shot 2017-02-15 at 13 48 38" src="https://cloud.githubusercontent.com/assets/5732209/22977403/64a7bf2c-f386-11e6-886a-b40ba6f52f07.png">
<img width="157" alt="screen shot 2017-02-15 at 13 48 52" src="https://cloud.githubusercontent.com/assets/5732209/22977404/64b0450c-f386-11e6-987a-2abd9da3596a.png">
<img width="59" alt="screen shot 2017-02-15 at 13 49 04" src="https://cloud.githubusercontent.com/assets/5732209/22977405/64b318fe-f386-11e6-8931-b7517921fe9c.png">
<img width="97" alt="screen shot 2017-02-15 at 13 49 13" src="https://cloud.githubusercontent.com/assets/5732209/22977406/64b93fa4-f386-11e6-9cd2-fea2ff5772cd.png">
<img width="62" alt="screen shot 2017-02-15 at 13 49 22" src="https://cloud.githubusercontent.com/assets/5732209/22977407/64ba7496-f386-11e6-9463-af034252f966.png">
<img width="87" alt="screen shot 2017-02-15 at 13 49 32" src="https://cloud.githubusercontent.com/assets/5732209/22977408/64bc5c98-f386-11e6-824d-569794ad92bc.png">
<img width="126" alt="screen shot 2017-02-15 at 13 49 43" src="https://cloud.githubusercontent.com/assets/5732209/22977410/64be38b0-f386-11e6-9f08-973b5bb9f7b0.png">
<img width="97" alt="screen shot 2017-02-15 at 13 49 54" src="https://cloud.githubusercontent.com/assets/5732209/22977411/64c5e90c-f386-11e6-9646-f0305d5ab001.png">
<img width="61" alt="screen shot 2017-02-15 at 13 50 03" src="https://cloud.githubusercontent.com/assets/5732209/22977412/64cecbda-f386-11e6-83f9-e9cf8482370b.png">
<img width="95" alt="screen shot 2017-02-15 at 13 50 18" src="https://cloud.githubusercontent.com/assets/5732209/22977414/64d1a5bc-f386-11e6-8724-60eb03987bcc.png">
<img width="307" alt="screen shot 2017-02-15 at 13 51 26" src="https://cloud.githubusercontent.com/assets/5732209/22977413/64d14ad6-f386-11e6-878d-3a219a8ec590.png">
<img width="66" alt="screen shot 2017-02-15 at 13 51 37" src="https://cloud.githubusercontent.com/assets/5732209/22977415/64d3d0d0-f386-11e6-898e-921ebeb2cfaf.png">
<img width="104" alt="screen shot 2017-02-15 at 13 51 48" src="https://cloud.githubusercontent.com/assets/5732209/22977416/64d94f6a-f386-11e6-8702-51b4bf1cbf96.png">
<img width="65" alt="screen shot 2017-02-15 at 13 52 08" src="https://cloud.githubusercontent.com/assets/5732209/22977417/64def6e0-f386-11e6-8dba-7b1e08702262.png">
<img width="61" alt="screen shot 2017-02-15 at 13 52 20" src="https://cloud.githubusercontent.com/assets/5732209/22977418/64e4568a-f386-11e6-99c7-cd28a085c740.png">
<img width="141" alt="screen shot 2017-02-15 at 13 52 38" src="https://cloud.githubusercontent.com/assets/5732209/22977419/64e9f072-f386-11e6-8df5-89344d7b4333.png">
<img width="67" alt="screen shot 2017-02-15 at 13 52 48" src="https://cloud.githubusercontent.com/assets/5732209/22977420/64ea5bfc-f386-11e6-922c-56f8fea4be0a.png">
<img width="100" alt="screen shot 2017-02-15 at 13 52 59" src="https://cloud.githubusercontent.com/assets/5732209/22977421/64f8b012-f386-11e6-9bb4-b81dd535dc25.png">
<img width="113" alt="screen shot 2017-02-15 at 13 53 39" src="https://cloud.githubusercontent.com/assets/5732209/22977422/64fa0f0c-f386-11e6-8804-5da9aa8af24e.png">
<img width="61" alt="screen shot 2017-02-15 at 13 54 34" src="https://cloud.githubusercontent.com/assets/5732209/22977424/650279ee-f386-11e6-9905-5a6ee29f16b7.png">
<img width="57" alt="screen shot 2017-02-15 at 13 54 55" src="https://cloud.githubusercontent.com/assets/5732209/22977423/64feefcc-f386-11e6-82cc-b6559df8ca9b.png">

**Comment by markwilliamfirth** [comment]
@nelsonic every one of those companies (bar one) writes their company name with an uppercase letter to begin.

The only brand that doesn't do it is eBay.

Most brand names and trademarks are capitalised

Examples of non-capitalised are easyJet, id Software, eBay, iPod, iPhone

**Comment by jruts** [comment]
Twitch is capitalized and Twitch is **awesome**. You should do whatever Twitch does.

**Comment by nelsonic** [comment]
@markwilliamfirth ***apologies***, the `main` thing I wanted to focus on was that if someone wanted to write `dWyL`, `dWYL`, `DWyL` or even `DWeeeL` or `DwiLL` I _really_ don't care as long as they are _opening_ and _replying_ to our emails!! 💌 

I acknowledged above that there are **_many_ more** examples of logos and brands that use uppercase. that's fine. our **end-users _don't care_**. they _care_ that our `product` solves their problem.

> it doesn't "_bug_" me in the _slightest_ if people write it "_incorrectly_" what bugs me is that nobody is creating _useful_ content or focussing on learning tools/skills to build the product. 

if we just use `dwyl` for marketing and `DWYL` for legal docs ***for now*** `while` we are _focussing_ on building the _product_ we can always re-visit this issue (_much_) later. 

once we have **1000 paying `product` customers** we'll have _full_ a branding discussion until then this is 

![image](https://cloud.githubusercontent.com/assets/194400/22977183/8a737d0a-f385-11e6-9ea4-e91d5999bc43.png)
 
let's _focus_ on figuring out how to get **feedback** from people. see: https://github.com/search?q=org%3Adwyl+feedback&type=Issues and _test_ the ideas/features we want to include in the product with real users. ❤️ 
literally ***everything*** `else` can wait until we have a `product`. ✅

**Comment by nelsonic** [comment]
Actually, ignore _anything_ I say on this. just go with what ever "_makes sense_". 👍 
(_but let's **move on** from this discussion, because I don't think the **people using** our app care._)

**Comment by markwilliamfirth** [comment]
@nelsonic The reason I wanted to settle this issue is because it should have be a relatively simple and quick decision and I'm of the opinion that 'DWYL' removes the issues associated with inconsistency and clarity that it's an acronym (reported in the issue). The reason we're still in discussion is because there was pushback that it looks `shouty` and `less friendly` and that `DWYL isn't an acronym`. I think these are internal perceptions that haven't been validated and instead `DWYL` is a logical choice that removes the issue at hand. The reason for my comment earlier today was to post evidence of an occurrence that happened today which shows the issue is ongoing. It might not be the biggest issue on our plate, but I don't understand the pushback when the solution seems quite simple. If you think people don't care or it's a distraction that doesn't matter then surely it doesn't matter if we change to using caps? That argument works both ways.

I agree that gathering feedback and outside perspectives would be useful in this situation, hence my posting of evidence of the ongoing issue earlier.

<img width="554" alt="screen shot 2017-02-15 at 16 39 05" src="https://cloud.githubusercontent.com/assets/5732209/22984453/4e04a46c-f39d-11e6-8acf-aa378531167a.png">

A quick message to a friend shows that **to them** `DWYL` isn't even completely clear that it's an acronym.

Not saying this should be our focus or we should spend a lot of time on this, but I think we should passively collect feedback from people and customers over time.

**Comment by iteles** [comment]
Agreed with passively collecting feedback. Are we going with `dwyl` for 3 months and  collecting feedback as suggested [above](https://github.com/dwyl/brand/issues/1#issuecomment-279099664)?

**Comment by iteles** [comment]
<img width="598" alt="screen shot 2017-02-15 at 16 49 40" src="https://cloud.githubusercontent.com/assets/4185328/22984922/d310281a-f39e-11e6-9430-1a390a743569.png">

😆

**Comment by markwilliamfirth** [comment]
@iteles `dwyl` has been used for over a year and we have evidence of this already (anecdotal and in emails etc) which validated the issue in the first place, hence my comment earlier enquiring about evidence. It would make more sense to try `DWYL` for 3 months and see if this has any effect and if necessary compare the number of errors compared to historical emails from the previous 3 months. Then we can make a decision on whether the brand consistency and customer feedback is valuable enough to make the change permanent or to switch back.

With regards to the message to Sue I think it's important to remember that whenever we introduce ourselves we always explain what DWYL stands for and other people normally latch onto that and use that as our name instead
>"Oh you know the Do What You Love guys?"
This potentially shows that DWYL is maybe either difficult to pronounce or explain to others and potentially not a good choice to begin with, regardless of casing.

I'd say either way we shouldn't make branding decisions based on our personal perceptions, but rather that they should be logical business decisions that help us create a strong brand.

**Comment by iteles** [comment]
I respectfully disagree. We haven't made any effort to quantify or qualify any evidence of how often `dwyl` is used vs any other variation, mostly because this hasn't been a concern.

Rather than a personal perception, lower casing is a well known user pattern for friendliness and is altogether miles more congruent with our brand.

**Comment by markwilliamfirth** [comment]
@iteles Ok so let's do 3 months of passively collecting evidence on here for `dwyl` and then 3 months of `DWYL`?

Whenever something occurs we'll just paste in a screenshot / quote?

**Comment by iteles** [comment]
Let's do 3 months of `dwyl` and see if there's a compelling reason to change this 👍

**Comment by markwilliamfirth** [comment]
Ok - let's record instances here as they occur and see how we go 👍
